Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
```

Comment:
@xu-cheng - Thoughts on Jack's comment? It's your code, I don't want to accidentally misrepresent your intent or anything, so I'll pass this to you directly.